src/main/scala/mesosphere/marathon/util/FaultDomain.scala

```diff
@@ -0,0 +1,4 @@
+package mesosphere.marathon
+package util
```

Comment:
I think it should be placed somewhere else, not in `util`. What do you think?

Reply:
completely agree with ivan! 
We should have a domain package... or a constraint package.. refactor the constraints into constraint package and add this.  
Util is a waste land... and should be lower level technical functions... not high order business functions / classes.

Reply:
Hm, this could be part of `state` or `mesosphere.marathon.core.instance`.